packages/yourloops/components/menus/team-scope-menu.tsx

```diff
@@ -0,0 +1,224 @@
+import { type TeamEditModalContentProps } from '../../pages/hcp/types'
+import { useAlert } from '../utils/snackbar'
+import GroupsOutlinedIcon from '@mui/icons-material/GroupsOutlined'
+import { useLocation, useNavigate } from 'react-router-dom'
+import TeamUtils from '../../lib/team/team.util'
+import useMediaQuery from '@mui/material/useMediaQuery'
+import IconButton from '@mui/material/IconButton'
+import Button from '@mui/material/Button'
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown'
+import Typography from '@mui/material/Typography'
+import MenuLayout from '../../layout/menu-layout'
+import MenuItem from '@mui/material/MenuItem'
+import ListItemIcon from '@mui/material/ListItemIcon'
+import ListItemText from '@mui/material/ListItemText'
+import Divider from '@mui/material/Divider'
+import AddIcon from '@mui/icons-material/Add'
+import TeamEditDialog from '../../pages/hcp/team-edit-dialog'
+
+const classes = makeStyles()((theme: Theme) => ({
+  sectionTitle: {
+    fontWeight: 'bold',
+    paddingLeft: theme.spacing(2),
+    paddingBottom: theme.spacing(1)
+  },
+  menu: {
+    paddingTop: theme.spacing(1),
+    paddingBottom: theme.spacing(1)
+  }
+}))
+
+const MENU_MAX_WIDTH_PX = 250
+const HOME_PATHNAME = '/home'
+
+export const TeamScopeMenu: FunctionComponent = () => {
+  const { t } = useTranslation('yourloops')
+  const {
+    classes: {
+      sectionTitle,
+      menu
+    }
+  } = classes()
+  const { getMedicalTeams, getPrivateTeam } = useTeam()
+  const { selectTeam, selectedTeamId } = useSelectedTeamContext()
+  const { createTeam } = useTeam()
+  const alert = useAlert()
+  const navigate = useNavigate()
+  const [teamCreationDialogData, setTeamCreationDialogData] = React.useState<TeamEditModalContentProps | null>(null)
+  const theme = useTheme()
+  const isMobile: boolean = useMediaQuery(theme.breakpoints.only('xs'))
+  const { pathname } = useLocation()
+
+  const privateTeam = getPrivateTeam()
+  const sortedMedicalTeams = TeamUtils.sortTeamsByName(getMedicalTeams())
+  const hasMedicalTeams = sortedMedicalTeams.length > 0
+  const availableTeams = [privateTeam, ...sortedMedicalTeams]
+
+  const selectedTeam = availableTeams.find((team: Team) => team.id === selectedTeamId)
+  const isSelectedTeamPrivate = selectedTeam.type === TeamType.private
+  const selectedTeamName = isSelectedTeamPrivate ? t('hcp-team-menu-private-practice-option') : selectedTeam.name
+
+  const privatePracticeIcon = <PersonIcon data-testid="private-practice-icon" />
+  const medicalTeamIcon = <GroupsOutlinedIcon data-testid="medical-team-icon" />
+  const selectedTeamIcon = isSelectedTeamPrivate ? privatePracticeIcon : medicalTeamIcon
+
+  const [anchorEl, setAnchorEl] = useState<HTMLElement | null>(null)
+  const isMenuOpen = !!anchorEl
+
+  const onSelectTeam = (teamId: string): void => {
+    if (teamId !== selectedTeamId) {
+      selectTeam(teamId)
+
+      if (pathname !== HOME_PATHNAME) {
+        navigate('/')
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Wouldn't it be better to navigate to `HOME_PATHNAME`?